src/gui.js

```diff
@@ -5533,6 +5571,14 @@ IDE_Morph.prototype.switchToScene = function (scene, refreshAlbum) {
     });
     scene.applyGlobalSettings();
     this.world().keyboardFocus = this.stage;
+
+    if (this.currentCategory != 'unified' && scene.unifiedPalette) {
```

Comment:
I feel like two checks is a little more than necessary -- this might be easier if we kept a `previousScene` but that didn't really feel worth it...